packages/react/src/themes/tokens.ts

```diff
@@ -2,10 +2,10 @@ import { main } from './main';
 import { Theme } from './theme';
 
 export const tokens: Theme['tokens'] = {
-    'focus-box-shadow': `0 0 0 2px ${main['primary-1.2']}`,
-    'focus-border-box-shadow': ` 0 0 0 1px ${main['primary-1.1']}, 0 0 0 3px ${main['primary-1.2']}`,
-    'focus-border-box-shadow-inset': `inset 0 0 0 2px ${main['primary-1.2']}, inset 0 0 0 3px ${main['primary-1.1']}`,
-    'focus-border': `${main['primary-1.1']}`,
+    'focus-box-shadow': `0 0 0 2px ${main['brand-20']}`,
```

Comment:
If the new color tokens are actually used, this will create a breaking change with theming as these values are not currently provided by the apps (neither from `theme-directory` or custom overrides).

Reply:
What I'm saying is that currently, apps that use the DS provide client themes that do not contain the new token values. As such, if these themes are not updated first, DS components that use the new tokens will always have the default Equisoft colors.

The client themes are defined in the [theme-directory](https://github.com/kronostechnologies/theme-directory) repo but most apps apply overrides on top of that. So unless I'm missing something, I think these will have to be updated before we can start using the design tokens in the components. Would it be preferable to fallback to the old colors as long as we're not fully transitioned to the new tokens?

I'm also wondering how we'll handle versioning.

Reply:
Didn't know the existence of another place with styles.
I'ver very confused, way do we have the design system and a theme repo? 
That's very confusing we need a documentation for all these.

Question:
does the clients use tokens?

note: I need to review this part and understand the use to adjust it.

Reply:
The design system provides the `Theme` interface and a default implementation (the Equisoft theme). Every client is then free to pass different values to override the default colors (client themes).

Now, both /connect and /plan use the DS and we wanted to have a common place to store the client themes, as many of our clients use both products. This is why theme-directory exists.

The theme repo is not part of the DS and it should probably not inform the decisions we make. But we still might want to provide a fallback during the transition to from basic colors to design tokens. That will give time to adapt their theming strategy while still allowing them to get DS updates.

I'll guess we'll discuss that more extensively in tomorrow's meeting! 😄

Reply:
I was thinking about it and maybe we can have a convertor function which will map the old colours name to the new ones for now until all the components get updated?
so no changes are done in the data folder until we make the full change.